### PR TITLE
added a way to save configuration files

### PR DESCRIPTION
**Pull request by gardnmi**
I came across your project on Reddit and found it extremely useful for creating workflows.  I currently use dbx to manage Databricks workflows and wanted to use your project to construct the workflows but not deploy them.

My changes introduce a new `create_config` option, allowing users to generate configuration files in JSON or YAML format. These configuration files will be added to the `workflow_definitions path.

### CODE REVIEW

**Comment by rafaelpierre** [comment]
Hey @gardnmi! First of all thanks for the comments - super nice that you made a PR :)

I was just curious about the use case - is the idea to generate YAML or JSON output files based on the Python definitions, and then use DBX to deploy the workflows?

I'll have a look at the PR in the meantime :)

File: examples/simple_workflow/config/my_workflow.json
Comment (rafaelpierre) [inline examples/simple_workflow/config/my_workflow.json:1]: Is this an example for the input? Curious about this one, as I expected DBX and/or the Jobs and Workflows REST API to be able to take this as an input

Comment (gardnmi) [reply examples/simple_workflow/config/my_workflow.json:1]: That is correct, right now in my dbx project I manage a hybrid jinja/python solution to generate the yml files required for dbx but it's very awkward to update them.  I want to leverage your project and define the workflow in a more pythonic manner.

**Comment by gardnmi** [comment]
Yes, I want to save the payload that this project generates to be sent to the jobs API as a yml or json file.  That way I can use pyjaws to create workflows and dbx to manage the deployment.

**Comment by rafaelpierre** [comment]
Gotcha, let me have another look at your PR. I think we can remove the output files from the examples section (since those are more for input .py scripts) and keep it in the docs and/or somewhere else

**Pull request closed without merging**